### PR TITLE
Replace invalid env with args in nodeserver

### PR DESCRIPTION
**Pull request by Haoning-Sun**
### What changes are proposed in this pull request?

The bash script entrypoint.sh  can not use FUSE_ALLUXIO_PATH and MOUNT_POINT env.

### Why are the changes needed?
Alluxio csi can not mount in fuse pod. 

### Does this PR introduce any user facing changes?
no

### CODE REVIEW

File: integration/docker/csi/alluxio/nodeserver.go [hunk 1]
```diff
@@ -246,8 +240,15 @@ func getAndCompleteFusePodObj(nodeId string, req *csi.NodeStageVolumeRequest) (*
...
 	fuseOptsStr := strings.Join(req.GetVolumeCapability().GetMount().GetMountFlags(), ",")
 	csiFusePodObj.Spec.Containers[0].Args = append(csiFusePodObj.Spec.Containers[0].Args, "--fuse-opts=" + fuseOptsStr)
-        csiFusePodObj.Spec.Containers[0].Args = append(csiFusePodObj.Spec.Containers[0].Args, req.GetStagingTargetPath())
-        csiFusePodObj.Spec.Containers[0].Args = append(csiFusePodObj.Spec.Containers[0].Args, targetPath)
+
+	// Set fuse mount point
+	csiFusePodObj.Spec.Containers[0].Args = append(csiFusePodObj.Spec.Containers[0].Args, req.GetStagingTargetPath())
+
+	// Set alluxio path to be mounted if set
+	alluxioPath := req.GetVolumeContext()["alluxioPath"]
```
Comment (ssz1997) [inline integration/docker/csi/alluxio/nodeserver.go:248]: With the new changes to `entrypoint.sh` and `alluxio-fuse`, if `alluxioPath` is not set in the Fuse k8s yaml file,  `alluxio.fuse.mount.alluxio.path` property shall be used for the `alluxioPath`. So we are not going to set it to `/` if `alluxioPath` is not set here.

@Haoning-Sun FYI

**Review by ssz1997 — APPROVED** [review]
Thanks for fixing this!

**Comment by ssz1997** [comment]
@HelloHorizon Let's merge this PR

**Comment by HelloHorizon** [comment]
alluxio-bot, merge this please

**Pull request merged**